setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.2.1
+current_version = 2.3.0
```

Comment:
I think 2.2.2 is fine. I would consider this a patch version since its backwards compatible and has no visible effects on the SDK from a user perspective